.speakeasy/workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflowVersion: 1.0.0
+sources:
+    openapi:
+        inputs:
+            - location: ../../apps/web/openapi.json
+        overlays:
+            - location: https://raw.githubusercontent.com/dubinc/dub-node/main/codeSamples.json
+        output: ../../apps/web/openapi_with_sdks.json
```

Comment:
output location should be from the root directory, is that correct?

Reply:
Thats right this in the `apps` root directory